README.md

```diff
@@ -560,6 +562,10 @@ Yamale uses [Tox](https://tox.readthedocs.org/en/latest/) to run its tests again
 versions. To run tests, first checkout Yamale, install Tox, then run `make test` in the Yamale's
 root directory. You may also have to install the correct Python versions to test with as well.
 
+NOTE on Python versions: If the current tox.ini specifies multiple versions of Python, consider
```

Comment:
Exactly! There are two versions specified in the tox.ini - and you need to be aware of this: If you're not, you won't install tox-pyenv (it is not mentioned anywhere else, but is a dependency for testing with multiple versions - unless there is another way of doing that about which I am unaware). Not everyone will be as new to Python and tox as I was (am), but I think it will still benefit from explicit suggestion/direction.

How about:

`Make test` validates functionality against the lowest and highest version of Python supported by Yamale. Unless your development environment is configured to support testing against multiple Python versions, one or more of the test branches may fail. One method of enabling testing against multiple versions of Python is to install `pyenv` and `tox-pyenv` and to use `pyenv install` and `pyenv local` to ensure that `tox` is able to locate appropriate Pythons.

Reply:
Sorry, I should have been clearer. When I suggested to "get rid of that caveat", I meant to remove the first part that says, "If the current tox.ini specifies multiple versions of Python". The rest of the sentence was fine. I'll try to do better to be more explicit. :) 

I'm fine with your new text as well. Either would help developers.

Reply:
Ah, gotcha!